python/lsst/ts/ataos/model.py

```diff
@@ -44,6 +44,21 @@ def __init__(self, log):
             "v": 0.0,
         }
 
+        # Hexapod sensitivity matrix, this is to account for cross terms
+        # between the axis. By default, there are no cross terms. This can be
+        # used, for instance, to add automatic tip-tilt correction for a given
+        # hexapod translation (x/y).
+        self._hexapod_sensitivity_matrix = np.array(
```

Comment:
this comes from the config, no? why is it defined here?

Reply:
Also, I think we should define a test for this, where the cross-terms are non-zero of course, no?

Reply:
So, this is supposed to be an independent class that can run and test without the CSC. The configuration belongs to the CSC business logic, which this class has no knowledge about. 

I can't remove these default values. What I can do is remove the defaults from the configuration and make the CSC not set them if the user does not provide some values. Nevertheless, it also means users won't have anywhere to look for an example of the configuration. Which maybe could be done in documentation.